src/dom/getancestors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * Returns all ancestors of given DOM node, starting from the top-most (root). Includes the given node itself.
+ *
+ * @param {Element|Text} node DOM node.
+ * @returns {Array.<Node>} Array of given `node` parents.
```

Comment:
Doesn't it return elements only? Or can it also be a docfrag? In any case, the documentation could be more precise.